frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/result-tab/result-tab-row.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Expandable } from '@patternfly/react-core';
+
+import './result-tab-row.scss';
```

Comment:
I have already raised following question in another PR but it was not answered.

Throughout the code, I can see import of SCSS
- like this one
- like this one but filename is prefixed with underscore `_` (like it should be used via SCSS `@import` but is not)
- single hierarchy built via SCSS `@import`initiated by `style.scss` for both `@console/internal` and `kubevirt-plugin`

Both cases have there pros and cons (encapsulation vs. single understandable hierarchy). 
Considering we are developing an end-user application and not a reusable library, the case of a single hierarchy makes more sense to me. But I do not have a strong opinion on that as long we stay consistent.

@vojtechszocs , @rawagner , @suomiy , wdyt?

Reply:
my preference is to have it at the same place as the `tsx` file and maybe have one place for common css files

Reply:
This is not a blocker for this PR, so let's discuss that face to face. The point is just to be consistent.